src/modules/TextOverlay/TextOverlay.js

```diff
@@ -0,0 +1,25 @@
+module.exports = exports = function(pixels, options,priorstep){
+    var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+
+    options.text = options.text || defaults.text;
+    options.x = options.x || defaults.x;
+    options.y = options.y || defaults.y;
+    options.font = options.font || defaults.font;
+    options.color = options.color || defaults.color;
+    options.size = options.size || defaults.size;
+
+
+        var img = $(priorstep.imgElement);
```

Comment:
This is only available in a browser so you will have to check for the inBrowser flag and throw a notification if a browser isn't available. This is why travis is failing. Travis doesn't have a browser, it works in node.

Reply:
Ohh that's really smart . This never occured to me !

Reply:
Thanks a lot @HarshKhandeparkar !! This is really helpful